standard/flags_master_data.lua

```diff
@@ -21,1151 +21,1456 @@
 --   Other
 local data = {
 	-- ISO 3166-1 alpha-2
-	['andorra'] = {
-		name = 'Andorra',
-		flag = 'File:ad_hd.png'
-	},
-	['unitedarabemirates'] = {
-		name = 'United Arab Emirates',
-		flag = 'File:ae_hd.png'
-	},
 	['afghanistan'] = {
+		flag = 'File:af_hd.png',
+		localised = 'Afghan',
 		name = 'Afghanistan',
-		flag = 'File:af_hd.png'
-	},
-	['antiguaandbarbuda'] = {
-		name = 'Antigua and Barbuda',
-		flag = 'File:ag_hd.png'
-	},
-	['anguilla'] = {
-		name = 'Anguilla',
-		flag = 'File:ai_hd.png'
 	},
 	['albania'] = {
+		flag = 'File:al_hd.png',
+		localised = 'Albanian',
 		name = 'Albania',
-		flag = 'File:al_hd.png'
 	},
-	['armenia'] = {
-		name = 'Armenia',
-		flag = 'File:am_hd.png'
+	['algeria'] = {
+		flag = 'File:dz_hd.png',
+		localised = 'Algerian',
+		name = 'Algeria',
+	},
+	['americansamoa'] = {
+		flag = 'File:as_hd.png',
+		localised = 'American Samoan',
+		name = 'American Samoa',
+	},
+	['andorra'] = {
+		flag = 'File:ad_hd.png',
+		localised = 'Andorran',
+		name = 'Andorra',
 	},
 	['angola'] = {
+		flag = 'File:ao_hd.png',
+		localised = 'Angolan',
 		name = 'Angola',
-		flag = 'File:ao_hd.png'
+	},
+	['anguilla'] = {
+		flag = 'File:ai_hd.png',
+		localised = 'Anguillan',
+		name = 'Anguilla',
 	},
 	['antarctica'] = {
+		flag = 'File:aq_hd.png',
+		localised = 'Antarctic',
 		name = 'Antarctica',
-		flag = 'File:aq_hd.png'
+	},
+	['antiguaandbarbuda'] = {
+		flag = 'File:ag_hd.png',
+		localised = 'Antigua and Barbuda',
+		name = 'Antigua and Barbuda',
 	},
 	['argentina'] = {
+		flag = 'File:ar_hd.png',
+		localised = 'Argentine',
 		name = 'Argentina',
-		flag = 'File:ar_hd.png'
 	},
-	['americansamoa'] = {
-		name = 'American Samoa',
-		flag = 'File:as_hd.png'
+	['armenia'] = {
+		flag = 'File:am_hd.png',
+		localised = 'Armenian',
+		name = 'Armenia',
+	},
+	['aruba'] = {
+		flag = 'File:aw_hd.png',
+		localised = 'Aruban',
+		name = 'Aruba',
 	},
 	['ascensionisland'] = {
+		flag = 'File:ac_hd.png',
+		localised = 'Saint Helenian',
```

Comment:
Is this correct?

Reply:
MR Google:
> There is no indigenous population on Ascension. Instead, the population is made up of employees of the organisations operating on the island. These mainly consist of St Helenians (known as Saints), British and Americans although nationalities the world over can be found on the island.

so imo the best is `'Saint Helenian'`